bin/fuel-core/src/cli/run.rs

```diff
@@ -306,7 +309,7 @@ impl Command {
             profiling: _,
         } = self;
 
-        info!("Metrics config: {}", metrics);
+        info!("Metrics config: {:?}", metrics);
```

Comment:
A nit:
This will produce a confusing message in logs, for example:
```
Metrics config: [TxPool, Producer]
```
-or-
```
Metrics config: [All]
```

One may think that it shows modules that are enabled.

Reply:
Good catch=) Pached https://github.com/FuelLabs/fuel-core/pull/2310/commits/5aea57a2e325ba4c246d44717a692e87bd3d4dee